server/api/controllers/article.js

```diff
@@ -580,3 +579,120 @@ export const getArticle = async (req, res) => {
     });
   }
 };
+
+/**
+ * @export
+ * @function updateArticle
+ * @param {Object} req - request received
+ * @param {Object} res - response object
+ * @returns {Object} JSON object (JSend format)
+ */
+export const updateArticle = async (req, res) => {
+  try {
+    const {
+      body,
+      params: {
+        articleId
+      },
+      user: {
+        id: userId
+      }
+    } = req;
+
+    const foundArticle = await Article.findOne({
+      where: {
+        id: {
+          [Op.eq]: articleId
+        },
+      }
+    });
+
+    if (!foundArticle) {
+      return res.status(404).send({
+        status: 'fail',
+        message: 'Article not found'
+      });
+    }
+    const articleAuthor = await Article.findOne({
+      where: {
+        id: {
+          [Op.eq]: articleId
+        },
+        authorId: {
+          [Op.eq]: userId
+        }
+      }
+    });
+    if (!articleAuthor) {
+      return res.status(401).send({
+        status: 'fail',
+        message: "Sorry you can't edit this article"
+      });
+    }
+    let newArticle = await foundArticle.update(
+      body, { returning: true, plain: true }
+    );
+
+    newArticle = newArticle.toJSON();
+
+    return res.status(201).send({
```

Comment:
@oyedejipeace please change the `status code` to `200` and also the words of the message returned in the response to reflect the expected action. The intended action is to update an existing article not create a new one.